### ISSUE TITLE
Formatting of questions, guidance, example answers etc. when downloaded 

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2168]
Could we have more obvious formatting of the questions, guidance, example answers etc., so it’s visible what is what after this is downloaded? 

Currently all appears in the same size, and guidance and example answers heading appears in italics. 
It would be nice to have some 'visual' hierarchy if this is possible,
e.g. so questions would appear with bigger font in bold so it would be clearer what falls under that question
guidance - would be smaller font and maybe even the guidance text could appear in italics

**Comment by magdalenadrafiova** [comment]
Actually comments, guidance currently does not appear at all - will be nice to have the option when downloading a plan to have options to select whether the user wants to add the guidence, comments. etc

**Comment by dsisu** [comment]
In general, we should review the format of the default to make sure that it meets accessibility requirements. For readability, some funders require a minimum font size of 11.

Issue was raised during 2020 UX exercise:
Font size for plan downloads – default set to 9, too small, should be 11.

**Comment by pherterich** [comment]
This is potentially the same as #1453 as using a proper structure should also make a visual difference. There's a separate ticket for the guidance download (#2748) and we're addressing the font size issue locally (https://github.com/DigitalCurationCentre/DMPonline-Service/issues/193), thus I'll close this issue.
